### ISSUE TITLE
Component: Calendar, MultiSelect and Dropdown ARIA attributes is not present

### ISSUE THREAD

**Issue by mristov** [primefaces/primeng#14991]
### Describe the bug

![image](https://github.com/primefaces/primeng/assets/150306240/966f3081-3337-43b0-8fd6-ccfbaa23f1ea)
![image](https://github.com/primefaces/primeng/assets/150306240/d93abf4c-4b12-4589-bcdc-fd3bc1616ce2)

aria-expanded is missing in p-calendar, p-multiSelect and p-dropdown

Testing in **Accessibility Insights for Web** plugin in Chrome browser

### Environment

PrimeNG 17.10
Angular 17

### Angular version

17.0.5

### PrimeNG version

17.10

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

18.16.1
